examples/hello_world/pubspec.yaml

```diff
@@ -14,8 +14,11 @@ dependencies:
   vector_math: 2.0.8 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
 
 dev_dependencies:
+  flutter_driver:
+    sdk: flutter
   flutter_test:
     sdk: flutter
+  test: any
```

Comment:
We use concrete version numbers on `test`. You can copy the current number from another `pubspec.yaml` file.

Reply:
done.